tests/coverage/unreachable/debug-assert/expected

```diff
@@ -1,4 +1,10 @@
-coverage/unreachable/debug-assert/main.rs, 6, PARTIAL
-coverage/unreachable/debug-assert/main.rs, 7, PARTIAL
-coverage/unreachable/debug-assert/main.rs, 8, NONE
-coverage/unreachable/debug-assert/main.rs, 10, NONE
+Source-based code coverage results:
+
+tests/coverage/unreachable/debug-assert/main.rs (main)
+ * 10:1 - 10:11 COVERED\
+ * 11:9 - 11:10 COVERED\
+ * 11:14 - 11:18 COVERED\
+ * 12:30 - 12:71 UNCOVERED\
+ * 13:9 - 13:23 UNCOVERED\
```

Comment:
I'm surprised this one works as expected despite #3441.

Reply:
Indeed, it's quite surprising. I suppose it's due to the macros expanding to more elaborate code behind the scenes, and their spans being correctly propagated.